internal/distro/fedora33/distro.go

```diff
@@ -238,6 +243,11 @@ func sources(packages []rpmmd.PackageSpec) *osbuild.Sources {
 		fileSource := osbuild.FileSource{
 			URL: pkg.RemoteLocation,
 		}
+		if pkg.Secrets == "org.osbuild.rhsm" {
+			fileSource.Secrets = &osbuild.Secret{
+				Name: "org.osbuild.rhsm",
+			}
+		}
```

Comment:
Please explain why this code block is needed for Fedora distro and why it is not included in the `fedora32` distro definition.

Reply:
Because even Fedora systems can have subscriptions. E.g. to private Satellite repos or RHEL repos in case of cross-distro building.

Reply:
So why it is not add it also to F32? Because it is EOL?

Reply:
Sorry, I forgot about that part. Yes, it is EOL so I skipped it.